pkg/scheduler/algorithm/priorities/interpod_affinity.go

```diff
@@ -69,13 +72,19 @@ func (p *podAffinityPriorityMap) processTerm(term *v1.PodAffinityTerm, podDefini
 	if err != nil {
 		return err
 	}
+	if fixedNode.Labels == nil {
+		return nil
+	}
+
 	match := priorityutil.PodMatchesTermsNamespaceAndSelector(podToCheck, namespaces, selector)
-	if match {
-		for i, node := range p.nodes {
```

Comment:
This is the key why it can boost the performance. Thanks!

Reply:
Indeed.